### PR TITLE
Sambit/roger 1114

### PR DESCRIPTION
**Pull request by 04msambit**
@amitbose327 Please have a look.

Added 2 new fields to statasd_rogeros_tools_exec_time and statsd_rogeros_events : ( image_tag and tools_version )

### CODE REVIEW

File: bin/roger.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ def roger_version(root_dir):
                 version = f.read().strip()
     finally:
         print(version)
+    return version
```
Comment (ankanm) [inline bin/roger.py]: The `print(version)` line will end up printing the version all the time. Print it only when `-v` or `--version` is specified.

File: bin/roger.py [hunk 2]
```diff
@@ -81,7 +82,7 @@ def main():
         if sys.argv[1] == "-h" or sys.argv[1] == "--help":
             roger_help(root, commands)
         elif sys.argv[1] == "-v" or sys.argv[1] == "--version":
-            roger_version(root)
+            version = roger_version(root)
```
Comment (ankanm) [inline bin/roger.py]: Possibly move that `print(version)` here.

File: cli/utils.py [hunk 3]
```diff
@@ -10,6 +10,11 @@
 import time
 import json
 
+own_dir = os.path.dirname(os.path.realpath(__file__))
+root = os.path.abspath(os.path.join(own_dir, os.pardir))
+sys.path.insert(0, root + "/bin")
+from roger import roger_version
```
Comment (ankanm) [inline cli/utils.py]: I don't think it's a good idea to import roger from bin. Can you move that method into this file instead?

Comment (amitbose327) [inline cli/utils.py]: +1

**Comment by amitbose327** [comment]
LG apart from Ankan's comments

File: cli/utils.py (hunk 3 above)
Comment (04msambit) [inline cli/utils.py]: Moved the method to utils. Made necessary changes.

File: bin/roger.py (hunk 2 above)
Comment (04msambit) [inline bin/roger.py]: Done.

File: bin/roger.py (hunk 1 above)
Comment (04msambit) [inline bin/roger.py]: Moved to print_help.

**Comment by ankanm** [comment]
LGTM

**Pull request merged**